BattleTank/Input/TankControllerState.cs

```diff
@@ -12,10 +12,11 @@ public struct TankControllerState
         public bool PlantMine { get; }
         public TankControllerState(float moveY, float moveX, bool fire, bool speedBoost, bool plantMine) : this()
         {
-            if (Math.Abs(moveY) > 1)
-                throw new ArgumentOutOfRangeException(nameof(moveY), moveY, "Wartość musi byc w zakresie -1 <= Y <= 1");
-            if (Math.Abs(moveX) > 1)
-                throw new ArgumentOutOfRangeException(nameof(moveX), moveX, "Wartość musi byc w zakresie -1 <= X <= 1");
+            //Czy to jest konieczne?
+           // if (Math.Abs(moveY) > 1)
+           //     throw new ArgumentOutOfRangeException(nameof(moveY), moveY, "Wartość musi byc w zakresie -1 <= Y <= 1");
+           /// if (Math.Abs(moveX) > 1)
+             //   throw new ArgumentOutOfRangeException(nameof(moveX), moveX, "Wartość musi byc w zakresie -1 <= X <= 1");
```

Comment:
A w jaki inny sposób chcesz okreslić maksymalne wychylenie gałek w kontrolerze? Jeśli tego nie będzie, to kiedyś się okaże, żę nagle nasze czołgi w specyficznych momentach jeżdżą szybciej niż powinny. Jeśli znasz lepszy sposób na to, aby ograniczyć max MoveY i MoveX to zapoda, ale tutaj będę się upierał, ze to musi być. Czemu Ci to nie pasuje?

Reply:
Przeszkadza mi to przy wdrożeniu DPad, lecz to jest półśrodek